core/src/main/scala/cats/data/OptionT.scala

```diff
@@ -24,6 +39,14 @@ final case class OptionT[F[_], A](value: F[Option[A]]) {
   def cata[B](default: => B, f: A => B)(implicit F: Functor[F]): F[B] =
     fold(default)(f)
 
+  /**
+   * Effectful catamorphism on the Option. This is identical to [[foldF]], but it only has
+   * one parameter list, which can result in better type inference in some
+   * contexts.
+   */
+  def cataF[B](default: => F[B], f: A => F[B])(implicit F: FlatMap[F]): F[B] =
```

Comment:
I am aware of the notions of _catamorphisms_, but is that also a unified term or notion across `cats`? I can only grep the words  `def cata` in `OptionT`and in `CoFree`. 

Unless we want to extend the use across the whole of `cats`, perhaps we should drop them, to avoid confusion.

Reply:
There's a `cata` to `fold` already, so when seeing `foldF`, I'd expect `cataF` there as well. Whether `cata` should have been introduced with that name in the first place seems to be outside of the scope of this PR…but that's just my two cents.